README.md

```diff
@@ -1,13 +1,46 @@
-# Software Product Health Analyzer (SPHA)
+![SPHA Logo](docs/img/Software_Project_Health_Assistant_Secondary-Logo.png)
 
-## Code Style
+## About
 
-We use [ktfmt](https://github.com/facebook/ktfmt?tab=readme-ov-file) as a code style checker and reformater with the
-`kotlinlang` code style.
+SPHA is a fully automated tool suite that assesses and communicates all aspects
+of software product quality. It does so by combining data about your projects
+from sources like ticketing systems, and static analysis tools. For more details
+see [software-product.health](https://www.software-product.health).
 
-The [gradle plugin](https://github.com/cortinico/ktfmt-gradle#how-to-use-) is installed for all submodules through the
-conventions plugin. It adds two gradle tasks
-`./gradlew ktfmtCheck` and `./gradlew ktfmtFormat`. The gradle plugin is configured to use `kotlinlang`.
-ktfmt has an [IntelliJ plugin](https://plugins.jetbrains.com/plugin/14912-ktfmt) which automatically replaces IntelliJ's
-format command and applies ktfmt's formatting. The IntelliJ plugin must be configured to use `kotlinlang` under the
-plugin's settings. 
\ No newline at end of file
+## SPHA Library
+
+This project contains SPHA's core library. Its main purpose is to calculate a products health score
+based on a given `KpiHierarchy` and `RawValueKPIs`. Further, it provides the possibility to transform
+tool results into our internal `RawValueKPI` format. The transformation is handled by dedicated tool
+adapters.  
+With the [SPHA CLI Tool](https://www.github.com/fraunhofer-iem/spha-cli) we have created an executable tool using this
```

Comment:
Instead of "we have created" something like "there is a CLI tool" or "you can use the CLI tool"